admin_guide/scheduling/pod_placement.adoc

```diff
@@ -293,26 +293,30 @@ endif::[]
 . Create a project object that includes the
 `scheduler.alpha.kubernetes.io/node-selector` annotation and labels.
 +
+[source,yaml]
 ----
-{
-    "kind": "Namespace",
-    "apiVersion": "v1",
-    "metadata": {
-        "name": "ns1",
-        "annotations": {
-            "scheduler.alpha.kubernetes.io/node-selector": "env=test,infra=fedora" <1>
-        }
-    },
-    "spec": {},
-    "status": {}
-}
-
+apiVersion: v1
+kind: Namespace
+metadata
+  name: ns1
+  annotations: 
+    scheduler.alpha.kubernetes.io/node-selector: env=test,infra=fedora <1>
+spec: {},
+status: {}
 ----
 +
 <1> Annotation to create the labels to match the project label selector. Here, the key/value labels are `env=test` and `infra=fedora`.
++
+[NOTE]
+====
+When using the *Pod Node Selector* admission controller, you cannot use `oc adm new-project <project-name>` for setting project node selector. 
+Setting the project node selector using the `oc adm new-project myproject --node-selector='type=user-node,region=<region>` {product-title} sets 
```

Comment:
maybe s/Setting the project node selector using the *command* {product-title} sets/When you set the project node selector by using the *command* command, {product-title} sets